conf/defaults.ini

```diff
@@ -411,6 +411,10 @@ user_invite_max_lifetime_duration = 24h
 # Enter a comma-separated list of usernames to hide them in the Grafana UI. These users are shown to Grafana admins and to themselves.
 hidden_users =
 
+# Service account maximum expiration date in days.
+# Warning: Any existing tokens with days to live greater than this setting will be restricted to comply with this setting. This action is not reversible.
+service_account_expiration_day_limit = -1
```

Comment:
This does not belong under user settings. We should create a new section for service account and put the setting there. Also I think the name is a bit confusing, correct me if I am wrong but this setting is supposed to control max expiration time for tokens belonging to a service account?

Reply:
I agree, I believe I forgot the work token. However, I believe if I create a new entry for SA settings, the name can be simplified. Let me take a look at it.

Reply:
yes you can remove the service_account prefix

Reply:
I have renamed and relocated the variable as requested.